src/shared/components/ui/Button/Button.tsx

```diff
@@ -4,18 +4,20 @@ import { tv } from "tailwind-variants";
 import type { VariantProps } from "tailwind-variants";
 
 const buttonVariants = tv({
-  base: "focus-visible:ring-ring inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium text-white transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2",
+  base: "focus-visible:ring-ring inline-flex size-fit items-center justify-center whitespace-nowrap rounded-md font-medium text-white transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2",
   variants: {
     variant: {
       default: "bg-purple-500 hover:bg-purple-700",
       error: "bg-red-500 hover:bg-red-700",
       info: "bg-blue-500 hover:bg-blue-700",
       success: "bg-green-500 hover:bg-green-700",
+      outline:
+        "border border-purple-500 bg-transparent text-purple-500 hover:bg-purple-500 hover:text-white",
```

Comment:
Should the outline button have a filled hover?

nit: add active state to the buttons :)

Reply:
this buttons need further refinement, just copypasted a wellsync one to have an outline variant (not really focused in design here)